docs/guides/4-custom-rulesets.md

```diff
@@ -1,247 +1,102 @@
-Both of them support all the main JSONPath functionality and a little bit more, but this syntax may differ slightly from other JSONPath implementations.
-
-Your `given` value can be a string containing any valid JSONPath expression, or an array of expressions to apply a rule to multiple parts of a document.
-You can also consume your [aliases](#aliases) here if you have some defined.
+Rules are the most important part of a ruleset. For more details on rules and its properties, see [Rules](./4a-rules.md).
 
-Use the [JSONPath Online Evaluator](http://jsonpath.com/) to determine what `given` path you want.
+### Formats
 
-### Severity
+Formats are an optional way to specify which API description formats a rule, or ruleset, is applicable to. Currently Spectral supports these formats:
 
-The `severity` keyword is optional and can be `error`, `warn`, `info`, or `hint`.
+- `aas2` (AsyncAPI v2.x)
+- `aas2_0` (AsyncAPI v2.0.0)
+- `aas2_1` (AsyncAPI v2.1.0)
+- `aas2_2` (AsyncAPI v2.2.0)
+- `aas2_3` (AsyncAPI v2.3.0)
+- `aas2_4` (AsyncAPI v2.4.0)
+- `aas2_5` (AsyncAPI v2.5.0)
+- `oas2` (OpenAPI v2.0)
+- `oas3` (OpenAPI v3.x)
+- `oas3_0` (OpenAPI v3.0.x)
+- `oas3_1` (OpenAPI v3.1.x)
+- `json-schema` (`$schema` says this is some JSON Schema draft)
+- `json-schema-loose` (looks like JSON Schema, but no `$schema` found)
+- `json-schema-draft4` (`$schema` says this is JSON Schema Draft 04)
+- `json-schema-draft6` (`$schema` says this is JSON Schema Draft 06)
+- `json-schema-draft7` (`$schema` says this is JSON Schema Draft 07)
+- `json-schema-2019-09` (`$schema` says this is JSON Schema 2019-09)
+- `json-schema-2020-12` (`$schema` says this is JSON Schema 2020-12)
 
-The default value is `warn`.
-
-### Resolved
-
-By default, Spectral processes each rule on a "resolved" document (a file where
-all `$ref` JSON Schema references have been replaced with the objects they point
-to). While this is typically the desired behavior, there are some use cases
-where you may need to run a rule on the "raw" un-resolved document.
-
-For example, if you want to enforce conventions on the folder structure used for
-[splitting up
-documents](https://blog.stoplight.io/keeping-openapi-dry-and-portable?utm_medium=spectral&utm_source=github&utm_campaign=docs).
-
-If your rule needs to access the raw `$ref` reference values, you can set `resolved: false` to allow the rule to receive the raw un-resolved version of the document. Otherwise `resolved: true` is the default.
-
-Here's an example of a rule that can access `$ref` values:
+Specifying the format is optional, so you can completely ignore this if all the rules you are writing apply to any document you lint, or if you have specific rulesets for different formats. If you'd like to use one ruleset for multiple formats, the `formats` key is here to help.
```

Comment:
Maybe change "the formats key is here to help" to "use the formats key."?

Reply:
There was also some repetition here so I cleaned it up a bit.